process-engine/bpmn-studio
# ✨ Add BPMN Lint

**Changes:**

1. Add BPMN Lint.

**Issues:**

Closes #919

PR: #1354

**Examples:**

<img width="1412" src="https://user-images.githubusercontent.com/20394992/53077633-b8488080-34f2-11e9-9f41-9b015f268411.png">


<img width="1412" src="https://user-images.githubusercontent.com/20394992/53076308-b3360200-34ef-11e9-9eab-86bead218dbc.png">


**Other PRs:**

https://github.com/process-engine/bpmn-lint_rules/pull/1
https://github.com/process-engine/bpmn-lint_plugin/pull/1
https://github.com/process-engine/bpmn-io_custom-bundle/pull/13

## How can others test the changes?

1. Start BPMN-Studio.
2. Open any diagram.
3. Click the button in the upper left corner of the designer.
4. See warnings and/or errors.

(Note that errors indicate mistakes that will prevent this process from running on the PE)

## PR-Checklist

Please check the boxes in this list after submitting your PR:
